doc-github-rules.md

```diff
@@ -77,14 +81,32 @@ Once a PR has received sufficient review and concerns are satisfied, a designate
 ## Directly Editing Artifacts
 Persons with write access permission or higher can directly edit documents and push changes. This practice should be used very sparingly, and only in the draft stages of documents or to make trivial changes in documents already in use.
 
->**Making direct edits to "released" documents which could impact downstream consumers, without proper review phases, short-circuits the normal community process and can result in breaking changes and significant disruption.**
-> 
+> [!WARNING]
+> Making direct edits to "released" documents which could impact downstream consumers, without proper review phases, short-circuits the normal community process and can result in breaking changes and significant disruption.
+
+
+## Use checklists for large issues or PRs
+
+When working on a complex issue or PR, it is often preferable to use a checklist within one issue or PR. This avoids the use of multiple issues or PRs that could be difficult to track.  
+
+Especially with an issue related to a feature, all of the pull requests used to implement the feature can refer to the same issue. The progress made can be easily understood by looking at which boxes are checked and which are not. The status is found in one place.
+
+Github even uses this checklist system to print out a small progress summary when looking at the lists of open issues.
+
+You enter the checklist in the comment when you create the issue or the PR. To create a checklist, you use a markdown format similar to the one shown in the example below.
```

Comment:
What do you do with the checklist as things progress?

Reply:
You check mark the listed work item when completed. I can add this info in the article.

Reply:
Added NOTE to clarify.